Cargo.toml

```diff
@@ -15,6 +15,7 @@ name = "bdkffi"
 
 [dependencies]
 bdk = { version = "0.22", features = ["all-keys", "use-esplora-ureq", "sqlite-bundled"] }
+bitcoin = { version = "0.28.1", features = ["use-serde", "base64", "rand"] }
```

Comment:
Not sure we need this import? At least on my side I was able to re-use `bdk::bitcoin` without an issue.

Reply:
Ah yes good catch. My IDE was showing an error for some reason on the `Script` type but it's totally an IDE bug. In the process of looking into this I had tried to import the type directly from the bitcoin crate to see if it would fix it but then forgot to remove it.